src/pages/settings/Payments/AddPayPalMePage.js

```diff
@@ -100,7 +101,7 @@ class AddPayPalMePage extends React.Component {
                         onPress={this.setPayPalMeUsername}
                         pressOnEnter
                         style={[styles.mt3]}
-                        isDisabled={this.state.payPalMeUsername === this.props.payPalMeData.description}
+                        isDisabled={(this.state.payPalMeUsername === this.props.payPalMeData.description) || _.isEmpty(this.state.payPalMeUsername.trim())}
```

Comment:
Not sure why this check is there.

Reply:
Can we remove it?

Reply:
If it's not used, yes.

Reply:
@Puneet-here Please remove unused code here, thanks!

Reply:
It is still useful when user navigates to this page using the url but I don't mind either way

Reply:
I'm confused but thats a very rare case, but still on removing @pecanoro Any thoughts?

Reply:
You can't add a second paypal account if the first one is already there, right? That would be the only case in which the check would be useful.

Reply:
Yes we can't add multiple PayPal accounts

Reply:
Please proceed with changes @Puneet-here

Reply:
No but we can update it. Please check the SS @pecanoro the button says update paypal if we removed the logic the button will stay enabled without making any change after visiting the url
<img width="376" alt="Screenshot 2022-11-01 at 10 25 42 PM" src="https://user-images.githubusercontent.com/101883770/199291371-5f0e019a-f63c-41eb-8fea-9d340c491dc0.png">

Reply:
Remove the whole update flow.

Reply:
Just Adding is fine!